src/less/education-notice/education-notice.less

```diff
@@ -0,0 +1,69 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+
+.education-notice {
+    .background-color-token(education-notice-default-background-color, color-background-secondary);
+    .border-radius-token(education-notice-border-radius, border-radius-100);
+    color: var(--color-foreground-primary);
+    font-size: var(--font-size-default);
+    padding: var(--spacing-300);
+}
+
+.education-notice--prominent {
+    .background-color-token(education-notice-education-background-color, color-background-education);
+}
```

Comment:
How did this work on the previous section notice? I noticed we didn't have a prominent version.

Reply:
Where are you looking? We do currently have a prominent version, in Skin anyway...

<img width="1324" alt="image" src="https://github.com/eBay/skin/assets/1675667/10959e98-909e-4b65-bd1f-ad63acf940f6">

Reply:
I think the confusing part was that we had no prominent modifier
I see now that prominent version is toggled through the education notice class 
It's fine to leave then as is